config/custom_items.txt

```diff
@@ -933,3 +933,13 @@ item_icon: pwcd
 item_desc: This wrist mounted computer was the Belters' answer for the older PDAs at the time, seeing use throughout the golden era when Ceres still operated as a mining facility. It now still sees wide use on various civilian to government vessels by use of engineering, or mining departments to the rare science department. Having a strong reputation for being modular, this one comes with an alarm clock and radio but the geiger counter seems to have been busted due to time. 
 req_titles: Engineer, Prospector, Bartender, Crewman, Passenger, Corporate Liaison, Loss Prevention Associate, Senior Engineer, Roboticist, Chief Engineer, Biomechanical Engineer 
 }
+{
+ckey: rodmuth
+character_name: Degbert Kentbritt
+item_path: /obj/item/clothing/suit/storage/toggle/bomber
+item_name: Expensive Suit
```

Comment:
this is an improper noun, so `expensive suit`

it's also not a suit, just a jacket, so `expensive suit jacket` or `expensive jacket`, whichever is preferable